test/acceptance/graphql_resolvers/local_aggregate_hybrid_search_test.go

```diff
@@ -99,4 +99,35 @@ func aggregationWithHybridSearch(t *testing.T) {
 			},
 		})
 	})
+
+	t.Run("with grouping, sparse search and nearText", func(t *testing.T) {
+		query := `
+		{
+			Aggregate {
+				Company
+    			(
+				  	groupBy: "name"
+				  	hybrid: {
+						alpha: 0.5
+        				query: "Google"
```

Comment:
does it make sense to have `query` together with `searches.nearText`? are those parameters permitted to be used together?